paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -326,6 +330,10 @@ public class PaparazziPlugin : Plugin<Project> {
     return defaultConfig.targetSdkVersion?.apiLevel?.toString()
       ?: DEFAULT_COMPILE_SDK_VERSION.toString()
   }
+
+  private inline fun <reified T : AndroidVariantTask> Project.findTask(variant: BaseVariant): List<T> {
+    return tasks.withType(T::class.java).filter { it.variantName == variant.name }
+  }
```

Comment:
nit: i'd inline/remove this, since it's probably not going to see additional usage (and shouldn't long-term?)